drag_and_drop_v2/utils.py

```diff
@@ -5,3 +5,24 @@
 # Make '_' a no-op so we can scrape strings
 def _(text):
     return text
+
+
+def ngettext(text_singular, text_plural, n):
+    if n == 1:
+        return text_singular
+    else:
+        return text_plural
+
+
+class FeedbackMessages(object):
+    FINAL_ATTEMPT_TPL = _('Final attempt was used, highest score is {score}')
+    MISPLACED_ITEMS_RETURNED = _('Misplaced items were returned to item bank.')
+
+    CORRECTLY_PLACED_SINGULAR_TPL = _('Correctly placed {correct_count} item.')
+    CORRECTLY_PLACED_PLURAL_TPL = _('Correctly placed {correct_count} items.')
```

Comment:
@e-kolpakov Currently we just assemble them manually, but the plan is to eventually build some generic xblock gettext-based cli tools that will scan xblock packages and create appropriate po files.